qbft/spectest/tests/controller_spectest.go

```diff
@@ -130,7 +129,7 @@ func (test *ControllerSpecTest) testBroadcastedDecided(
 			r2, err := runData.ExpectedDecidedState.BroadcastedDecided.GetRoot()
 			require.NoError(t, err)
 
-			if bytes.Equal(r1, r2) &&
+			if r1 == r2 &&
```

Comment:
better use bytes.equal as far as I know

Reply:
idk if that is true in this case as r1 and r2 are not byte slices, they are arrays now. So to use bytes.Equal we would have to slice them like `bytes.Equal(r1[:], r2[:])`

Reply:
Yes, we can use `==` for array comparison. I think converting to a slice is redundant